Sources/PostgreSQL/Message+Parse/PostgreSQLMessageDecoder.swift

```diff
@@ -26,8 +26,17 @@ final class PostgreSQLMessageDecoder: ByteToMessageDecoder {
             return .needMoreData
         }
 
-        //// peek at messageSize
+        /// peek at messageSize
         guard let messageSize: Int32 = buffer.peekInteger(skipping: MemoryLayout<Byte>.size) else {
+            // Response from a PostgreSQLSSLSupportRequest will only return a single byte, so we need to handle that case
+            if (messageType == .S || messageType == .N), let data = buffer.readSlice(length: MemoryLayout<Byte>.size) {
+                let decoder = _PostgreSQLMessageDecoder(data: data)
+                let message = try PostgreSQLMessage.sslSupportResponse(decoder.decode())
+                ctx.fireChannelRead(wrapInboundOut(message))
+                VERBOSE("   [message=\(message)]")
+                return .continue
```

Comment:
I tend to avoid early return in an `if` (since there's no fall-through error). The `.needMoreData` case should be in an `else`.